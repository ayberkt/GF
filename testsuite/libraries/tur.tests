PhrUtt NoPConj (UttAdv (PrepNP by8agent_Prep (UsePron it_Pron))) NoVoc
PhrUtt NoPConj (UttAdv (PrepNP by8agent_Prep (UsePron she_Pron))) NoVoc
PhrUtt NoPConj (UttAdv (PrepNP by8means_Prep (UsePron he_Pron))) NoVoc
PhrUtt NoPConj (UttAdv (PrepNP by8means_Prep (UsePron it_Pron))) NoVoc
