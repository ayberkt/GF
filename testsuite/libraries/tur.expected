onun tarafından
onun tarafından
onun tarafından
onun tarafından
herkes şarkı söylüyor
